docs/INTERACTIVE_DEMO_RUNBOOK.md

```
@@ -0,0 +1,728 @@
+5. [Part 4: Agent Zero Integration](#part-4-agent-zero-integration-3-min) (3 min)
+6. [Part 5: Advanced Features](#part-5-advanced-features-5-min) (5 min)
+7. [Troubleshooting](#troubleshooting)
+
+---
+
+## Prerequisites
+
+Before starting the demo, ensure the following:
+
+### Required Services
+| Service | Port | Health Check |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)